frontend/test/lit/test_quantum_control.py

```diff
@@ -29,16 +30,16 @@
 def get_custom_qjit_device(num_wires, discarded_operations=None, added_operations=None):
     """Generate a custom device with the modified set of supported gates."""
 
-    lightning = qml.device("lightning.qubit", wires=3)
-    config = device_get_toml_config(lightning)
+    lightning = qml.device("lightning.kokkos", wires=3)
```

Comment:
It just uses kokkos list of operations because the one from lightning is a frozen set.